src/Components/Attributes/Attributes.js

```diff
@@ -105,7 +109,17 @@ class Attributes extends React.Component {
     // `dataStructures` prop is optional and is used to
     // resolve inheritance, references and includes/mixins, plus to
     // render the inheritance tree.
-    const dataStructures = props.dataStructures || [];
+    let dataStructures = props.dataStructures || [];
+    let element = props.element;
+
+    if (isFunction(element.getMetaProperty)) {
+      // serialize minim Element inputs (`element` and `dataStructures`)
+      // to the expected refract 0.6 serialization
+      const refractSerializer = new JSON06Serialiser(minim.namespace());
+      element = refractSerializer.serialise(element).content[0];
```

Comment:
As you are only caring about the first element inside the content, you could first grab that element before serialising so that you're not doing additional serialisations if there was further sub-elements.

```diff
- element = refractSerializer.serialise(element).content[0];
+ element = refractSerializer.serialise(element.get(0));
```

Reply:
Note this comment is invalid after my final review comment.